src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -75,17 +70,16 @@
  *
  * <p>1. To find the first caller filtering a known list of implementation class:
  * {@snippet lang="java" :
- *     StackWalker walker = StackWalker.getInstance(Kind.CLASS_INFO, Option.RETAIN_CLASS_REFERENCE);
+ *     StackWalker walker = StackWalker.getInstance(Option.DROP_METHOD_INFO, Option.RETAIN_CLASS_REFERENCE);
```

Comment:
Would this read better as "filtering **out** a known list of implementation **classes**" ?

Reply:
Yes - that's better.

Reply:
The example use filter so it might be better to have the text use the same word, I think "filtering out known implementation classes" works.